react-frontend/src/components/DigitalFramework/digitalblock.js

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import '../../css/digital.css';
+
+function DigitalBlock(props) {
```

Comment:
Is the naming of DigitalBlock or DigitalHeader signifcant? What makes it Digital?

Reply:
short for digitalframeworks page, but I can add the Framework word no problem